charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -27,7 +28,6 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/etcd-configmap: {{ include (print $.Template.BasePath "/etcd-configmap.yaml") . | sha256sum }}
```

Comment:
How do I calculate the checksum now? Any suggestions?

Reply:
See https://github.com/gardener/gardener/blob/744e0ff5db4f7f58d70a2f43546d379303f8a2cf/pkg/operation/seed/seed.go#L993

Reply:
While implementing, I have a doubt. Should we take the checksum of just the data (means `etcd.conf.yaml`) of the configmap or the whole configmap. If we want to take the checksum of whole configmap, it may be very difficult to test and I would also want to know why that is necessary. If we want to take checksum of just `etcd.conf.yaml` residing in data of the configmap, then I understand the reason